src/app/modules/group/pages/group-overview/group-overview.component.ts

```diff
@@ -4,7 +4,8 @@ import { GroupDataSource } from '../../services/group-datasource.service';
 @Component({
   selector: 'alg-group-overview',
   templateUrl: './group-overview.component.html',
-  styleUrls: [ './group-overview.component.scss' ]
+  styleUrls: [ './group-overview.component.scss' ],
+  providers: [ GroupDataSource ],
```

Comment:
That will create a new instance of the datasource for this component while the goal is the share it.

Reply:
fixed, I kept the provider in this component and removed the one in the `GroupOverviewComponent`